bin/mozetl-submit.sh

```diff
@@ -53,6 +55,7 @@ MOZETL_ARGS=${MOZETL_COMMAND:-$@}
 MOZETL_GIT_PATH=${MOZETL_GIT_PATH:-https://github.com/mozilla/python_mozetl.git}
 MOZETL_GIT_BRANCH=${MOZETL_GIT_BRANCH:-master}
 MOZETL_SPARK_MASTER=${MOZETL_SPARK_MASTER:-yarn}
+MOZETL_SUBMISSION_METHOD=${MOZETL_SUBMISSION_METHOD:=spark}
```

Comment:
This should be `:-` for consistency. I think it should also work, but has a slightly different behavior.

Reply:
Ok, thanks.  This is literally the first time I've ever set a default value in a bash script.  :)